### ISSUE TITLE
boot hang during kernel loading

### ISSUE THREAD

**Issue by scrax** [RevoGirl/RevoBoot#1]
I'm opening this discussion here about my problem at boot because I need to familiarize with github and also to leave here my findings.
Seems that the problem could be ralated to the boot0 and boot1h preloaders used.
I'm using one of the last Chameleon RC5 revision. FKA solved a similar problem using preloaders from Chameleon RC4.
I have not yet tried with other preloaders but I will update this post when done.

**Comment by RevoGirl** [comment]
Thanks you for reporting this. We need to have instruction for people to replace the two files. Most likely going to be added to the tree now that we became a stand-alone project.

**Comment by scrax** [comment]
Agree, If you can add a wiki to the project we can add some getting started info, and work on the documentation.

**Comment by scrax** [comment]
Good
I can see the edit page and new page buttons, but I have to go to work now, i'll be back this evening and i'll try to put something online.

**Comment by ghost** [comment]
Wow.. I'm busy during the weekend and I end up missing all the chat!.. lol :)

Can I just confirm that I'm using boot0 and boot1h from Chameleon RC5 trunk 699
http://forge.voodooprojects.org/p/chameleon/source/tree/HEAD/

EDIT: I see git recognised my reply here as closing this issue? Well that was not my intention...

**Comment by RevoGirl** [comment]
No worries. Reopening...

**Comment by humph** [comment]
On a similar note to blackosx, was away at weekend and find I've missed lots of exciting new progress. Now off to re-learn how git and this site works!
Oh, and I've always used RC4 r684 boot/boot1h files.

**Comment by RevoGirl** [comment]
I fixed a nasty memory allocation error in RevoBoot v1.0.07 so please give it a go.

**Comment by ghost** [comment]
Good find. 
V1.0.07 boots fine for me.

**Comment by RevoGirl** [comment]
I found a bug in the ACPI patcher that caused me a headache. I fixed it only recently, and after a long hunt for it, but now I can boot no matter what boot options I use. Either with a cold or a warm boot. Two very annoying bugs squashed, along with that boot file issue that has been resolved. Three issues fixed under the same umbrella and I am unaware of any other boot issues so please give it a go.

Note: You should no longer need to boot with Chameleon to get a working system Extensions.mkext

**Comment by scrax** [comment]
Hi Sam, have you seen my comment about the GAS fix on nvida chipset I'm using? It's a fix for intel chipset or cpu? I've tryed your patch posted on 23th without succes now I'll try this new modifications.

**Comment by humph** [comment]
Hi Sam, I presume you mean the change you posted at Insanely about XSDT. If so, I can report that it's fine on my atom notebook (but that was booting OK before), but alas the P55/i5 machine I was having ACPI problems with is now in storage. Am about to emigrate!  Sadly I guess by the time I see it again, we'll all be using some UEFI loader ;-)

**Comment by RevoGirl** [comment]
No. I found a bug in the ACPI patcher self which blocked my system, and others, from booting up properly.

And about to emigrate? Wow. To where, if you don't mind me asking (e-mail me if you want)? Hey. I am only going for a couple of months to another country, but I can't wait. Guess it must be like this times 1000 for you. Wish you all the best and please keep us posted!
